test/benchmark/core/Vector3Components.js

```diff
@@ -108,6 +108,8 @@
 
 		}
 
+		return result;
```

Comment:
Why is it necessary to return the result in the `add()` methods?

Reply:
Otherwise ESLint complains that `result` variable is unused.